### DISCUSSION TITLE
Remote Desktop tab missing

### DISCUSSION THREAD

**Discussion by Bauluzen** [Ylianst/MeshCentral#6533] [Issues] [answered]
Hello, I am encountering an issue with my setup. I have installed Mesh Commander within an isolated network and connected it to two Intel vPro devices via a switch. The setup appears to be correct; however, the 'Remote Desktop' tab is not available. When I try to connect to the desktop by right-clicking and selecting 'Connect to Desktop,' I receive the error message: 'This device indicated that this type of connection of not supported.' Can anyone help me resolve this issue?

**Comment by si458** [comment, marked as answer]
Not all devices support intel amt, I've just discovered that although the device has a label saying Pro, it only supports Intel ME and not AMT, so I can connect and manage the device using meshcommander, I can't use the kvm desktop features, 
You can check what amt u have by downloading the meshcmd and running `meshcmd.exe AmtInfo` and it will tell u what version/type you have

**Reply by Bauluzen** [reply]
You are right it says intel ME. Thank you for your quick response.

**Reply by si458** [reply]
@Bauluzen yes this is also what I discovered

**Comment by Bauluzen** [comment]
On the Intel website, it states that the i5-12500T processor supports Intel AMT. Could it be that I need to activate AMT again? I have already checked the option in the BIOS.

**Reply by si458** [reply]
Just because the processor supports amt, doesn't mean the motherboard/manufacturer enabled it!

I have 10 old i5 3rd gens, which support amt, yet the optiplexs have amt physically disabled on the motherboard and bios
But we actually have a single 1 out of the 10 that have the amt enabled and works no problems
